### PR TITLE
crypto (blake, ripe) jetted

### PR DESCRIPTION
**Pull request by BernardoDeLaPlaz**

### CODE REVIEW

File: jets/e/blake.c [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/* j/5/blake.c
+**
+*/
+#include "all.h"
+
+#include <argon2.h>
+#include "../src/blake2/blake2.h"
```
Comment (joemfb) [inline jets/e/blake.c]: Where is this header coming from? Should the PR include a new submodule?

File: jets/e/blake.c [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    return u3m_bail(c3__exit);
+  }
+
+  c3_w men_w;
+  men_w =  u3r_word(0, men);
+  
+  c3_y * mod_y =  (c3_y * ) u3a_malloc(men_w);
+  u3r_bytes(0, men,   (void *) mod_y, mod);
+
+  pretty_print_hex("mod_y (message body bytes)", mod_y, men);
```
Comment (joemfb) [inline jets/e/blake.c]: Please remove, or at least ifdef-out these debug printfs.

File: meson.build [hunk 3]
```diff
@@ -125,6 +126,7 @@ jets_e_src = [
 'jets/e/loss.c',
 'jets/e/lune.c',
 'jets/e/trip.c'
+'jets/e/ripe.c'
```
[the remark this answers is not in the document]
Comment (BernardoDeLaPlaz) [reply meson.build]: fixed

File: jets/e/blake.c (hunk 2 above)
Comment (BernardoDeLaPlaz) [reply jets/e/blake.c]: done

File: jets/e/blake.c (hunk 1 above)
Comment (joemfb) [reply jets/e/blake.c]: Ah, and that module is already added in the crypto-additions branch. Thanks!

Comment (BernardoDeLaPlaz) [reply jets/e/blake.c]: NP !

**Pull request merged**